tests/quantization/test_compressed_tensors.py

```diff
@@ -122,3 +126,45 @@ def test_compressed_tensors_w4a16_marlin24(vllm_runner):
         sampling_params = SamplingParams()
         output = llm.generate("Hello world!", sampling_params=sampling_params)
         assert output
+
+
+def test_compressed_tensors_kv_cache(tmp_path, vllm_runner):
+    # test the inference with the compressed-tensors specific
+    # quantized cache
+    model_path = snapshot_download(
```

Comment:
Instead of having this in code, why not just push a model to the hf_hub?

Reply:
Will do it on the final round of changes